src/main/java/org/mifos/integrationtest/config/ChannelConnectorConfig.java

```diff
@@ -34,6 +36,15 @@ public void setRequestType(String requestType) {
         this.requestType = requestType;
     }
 
+    public String getGsmaTransactionEndpoint() {
+        return gsmaTransactionEndpoint;
+    }
+
```

Comment:
you don't need a setter as it is injected automatically by Spring.  (@value)

Reply:
removed setter